svenfire99/project_bioscooop
# Ahmet

_Made auto folder structure create function that also makes a json file in it to save data. To test my pull request commit your changes by doing the following in the terminal:_

**- git status to see which files you have modified**
**- git add .**
**- git commit -m "your message about what you have done"**
**- git push origin HEAD**
**- git checkout Ahmet**

How to test my work:
_just run the code and look at the folder structure if a folder is added to the project folder and inside it there is an **SubFolder** folder and inside that there should be a file called MyNewFile.json. If that is all true then you can switch back to your own branch. to do that you just type in the terminal the following code._
**- git checkout _yourownbranchname_** 

_**PS: if you have any questions please make sure to contact me.**_
